### PR TITLE
Miscellaneous documentation improvements

### PR DESCRIPTION
**Pull request by capnrefsmmat**
See commit messages for descriptions of the changes. Basically I just randomly browsed the docs, saw a few things to improve, and improved them.

### CODE REVIEW

File: R/avail_endpoints.R [hunk 1]
```diff
@@ -1,11 +1,12 @@
-#' List all available endpoints.
+#' List all available Epidata API endpoints
 #'
-#' @description
-#' A function that prints a tibble with two columns: `Endpoint` contains the
-#' function for accessing the Delphi Epidata API endpoint along with a
-#' `Description`.
+#' Fetches a data frame of all Epidata API endpoints that can be accessed using
```
Comment (dsweber2) [inline R/avail_endpoints.R:3]: Isn't this missing a `@description`? Otherwise seems good

File: R/epidatacall.R [hunk 2]
```diff
@@ -103,42 +103,38 @@ print.epidata_call <- function(x, ...) {
...
-#'  is only used by `fetch_debug`, and by default is `"json"`
-#' @return
-#' - For `fetch_args_list`: a `fetch_args` object
+#' @param fields a list of epidata fields to return, or `NULL` to return all
+#'   fields (default). e.g. `c("time_value", "value")` to return only the
+#'   `time_value` and `value` fields or `c("-direction")` to return everything
+#'   except the direction field
+#' @param disable_date_parsing disable automatic date parsing; by default
+#'   `FALSE`
+#' @param disable_data_frame_parsing disable automatic conversion to data frame;
```
Comment (dsweber2) [inline R/epidatacall.R:119]: Just to confirm, linebreaks in roxygen only show up in the code itself, and not when the documentation is displayed, right? either way, not sure how this was at 87; maybe we were wrapping at 100?

File: R/epidatacall.R [hunk 3]
```diff
@@ -103,42 +103,38 @@ print.epidata_call <- function(x, ...) {
...
+#'   this is only supported by endpoints that only support the 'classic' format
+#'   (non-tabular). by default `FALSE`
```
Comment (dshemetov) [inline R/epidatacall.R:121]:
```suggestion
#'   (non-tabular); by default `FALSE`
```

**Review by dshemetov — APPROVED** [review]
Minor punctuation ocd, but otherwise LGTM, reads much better with your changes Alex!

**Review by dsweber2 — APPROVED** [review]
I'm a fan of being consistent about `@description`, but otherwise seems good; thanks for these!

File: R/epidatacall.R (hunk 3 above)
Comment (dsweber2) [reply R/epidatacall.R:121]: is a multi-semicolon a writing smell? :thinking:

File: R/avail_endpoints.R (hunk 1 above)
Comment (capnrefsmmat) [reply R/avail_endpoints.R:3]: Everything after the first line/paragraph is automatically in the Description section

File: R/epidatacall.R (hunk 3 above)
Comment (capnrefsmmat) [reply R/epidatacall.R:121]: Actually, this raises a question: do we need `@param` to list the defaults when they're already shown in the argument list? roxygen and R show the argument list, with defaults, at the top of the help. It's telling that a few of these were wrong, showing the flaw in repeating information in different places

File: R/epidatacall.R (hunk 2 above)
Comment (capnrefsmmat) [reply R/epidatacall.R:119]: Right, this all gets automatically reformatted when displayed. roxygen makes this a nice HTML list, while R I think automatically re-wraps lines based on your terminal width

File: R/epidatacall.R (hunk 3 above)
Comment (dshemetov) [reply R/epidatacall.R:121]: That's a fair point, looking around other packages, like [dplyr](https://dplyr.tidyverse.org/reference/filter.html), they tend to not mention defaults in the @param field.

File: R/avail_endpoints.R (hunk 1 above)
Comment (dsweber2) [reply R/avail_endpoints.R:3]: For my sanity, I'd rather include a `@description` if you don't mind. I've run into gotchas where the second paragraph automatically went into the details section, and so would rather just be explicit about sections.

**Comment by capnrefsmmat** [comment]
I've pushed some fixes based on the reviews. Also I switched to using `match.arg()` for an argument that can take on one of three options, because that's how all of base R does it, and it makes the available options visible in the argument list. (See, for example, the `alternative` argument to `t.test()`.)

**Comment by dshemetov** [comment]
Thanks again Alex. Also happy to learn about `math.arg()`, was wondering why people referred to those as choice arguments and not vector arguments 🙃

**Pull request merged**